spec/Section 6 -- Execution.md

```diff
@@ -15,13 +17,23 @@ A GraphQL service generates a response from a request via execution.
   being executed. Conceptually, an initial value represents the "universe" of
   data available via a GraphQL Service. It is common for a GraphQL Service to
   always use the same initial value for every request.
+- {onError} (optional): The _error behavior_ to apply to the request; see
```

Comment:
What about onError (or behavior) as a query directive rather than a new field on the request? Adding a field to the request payload might require changes to lots of GraphQL middleware

Reply:
This is where we initially started with `@disableErrorPropagation`. It's very handy that no middleware is impacted at all but @benjie [argumented that it should be a request parameter](https://github.com/graphql/nullability-wg/discussions/86).

The argument that won me is that the directive would most likely be automatically added by the Apollo/Relay/... compilers. Then it means you're modifying the user query and if the user types the same query in GraphiQL it won't behave the same, which is surprising.

Reply:
That's a good point that you might want to be able to add it without modifying the query document

Reply:
Also keep in mind that it being a GraphQL request parameter does not necessarily mean it’s an HTTP request parameter; for example you could use a different endpoint in HTTP to set the param: `/graphql_nobubblesplz` ;)